### PR TITLE
Better previews

### PR DESCRIPTION
**Pull request by endymion1818**
- [x] More tidying
- [x] Add images for backgrounds
- [x] Use `<select>` elements instead of radio buttons
- [x] Switch font sizes to specific settings
- [x] Use ternaries to display background and font sizes based on selection
- [x] Line up font sizes with background images
- [x] Print styles for large & small sizes
- [x] Tidy input & select fields
- [x] No background images — print styles remove them :-( 
- [x] SEOerize the app
- [x] Lines should only be printed when there is text in that line
- [x] Lines should be optional
- [x] Fix print layout

### CODE REVIEW

File: dot-print/index.html [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
         <div id="app">
             <div id="header">
-                <h1>Dot Printer</h1>
+                <h1>Handwriting practice paper</h1>
+                <!--googleoff: index-->
+                <p id="sorrynotresponsive">Sorry! You'll need to use a device with a larger screen for this to work.</p>
+                <!--googleon: index-->
+                <p>Make full-page custom handwriting worksheets in seconds!</p>
+                <ul id="instructions">
+                    <li><strong>STEP 1:</strong> Select a writing style: <i>cursive</i> or <i>d'nealian</i></li>
```
Comment (warrickhill) [inline dot-print/index.html]: Text here does not match the options in the drop down

**Review by warrickhill — CHANGES_REQUESTED** [review]
Looking good.

Some change requests: 
1. Lines should only be printed when there is text in that line
2. Lines should be optional

**Comment by endymion1818** [comment]
1. ~I've investigated that at length mate, believe me we have to choose between printing an image background to show lines, or making it part of the font, so it only shows up when you type a character. I'm going to try a trick with the inline-block property which is my last idea for that~ got it, will be in this PR
2. Ok I'll work on this.

File: dot-print/index.html (hunk 1 above)
Comment (endymion1818) [reply dot-print/index.html]: 👍  done ... d'nealean is apparently the standard way of teaching

**Pull request merged**